pkg/reportingplugins/mercury/v1/mercury.go

```diff
@@ -248,7 +249,40 @@ func parseAttributedObservation(ao ocrtypes.AttributedObservation) (PAO, error)
 		pao.PricesValid = true
 	}
 
-	if obs.CurrentBlockValid {
+	if len(obs.LatestBlocks) > 0 {
```

Comment:
probably also worth discarding an observation if it has _too many_ latest blocks